beard/clustering/wrappers.py

```diff
@@ -163,11 +169,13 @@ def fit(self, X, y=None):
         # As default value we use the highest so we obtain only 1 cluster.
         best_threshold = self.linkage_[-1, 2]
         n_clusters = self.n_clusters
-        scoring = self.scoring
+        supervised_scoring = self.supervised_scoring
+        unsupervised_scoring = self.unsupervised_scoring
         threshold = self.threshold
         ground_truth = (y is not None) and np.any(np.array(y) != -1)
+        scoring = supervised_scoring is None and unsupervised_scoring is None
```

Comment:
Use the negative value. The name `scoring` suggests that it is true if there is a scoring function, while it is actually the opposite as you have stated it...